LongWarOfTheChosen/Config/XComPromotionUIMod.ini

```diff
@@ -35,7 +35,7 @@ FactionAbilityCosts[3] = 12
 FactionAbilityCosts[4] = 13
 FactionAbilityCosts[5] = 14
 FactionAbilityCosts[6] = 25
-FactionAbilityCosts[7] = 25
+FactionAbilityCosts[7] = 30
```

Comment:
Because it was too easy and strong to dump all your XCOM AP in hero classes. If we make them have the same rank up rules as standard soldiers then this won't likely be needed.